utils/test/src/main/kotlin/Utils.kt

```diff
@@ -189,6 +189,12 @@ fun scannerRunOf(vararg pkgScanResults: Pair<Identifier, List<ScanResult>>): Sca
         )
     }
 
+    val scanners = mutableMapOf<Identifier, MutableSet<String>>()
```

Comment:
This can be simplified to:

```
val scanners = pkgScanResults.map { (id, scanResultsForId) ->
        id to scanResultsForId.mapTo(mutableSetOf()) { it.scanner.name}
    }.toMap()
``

Reply:
IntelliJ suggested even the further optimization:

```
val scanners = pkgScanResults.associate { (id, scanResultsForId) ->
        id to scanResultsForId.mapTo(mutableSetOf()) { it.scanner.name }
}
```